hacks_mbof/urls.py

```diff
@@ -43,7 +45,20 @@
     url(r'^bower_components/(?P<path>.*)$', serve, {
         'document_root': 'mbofui/bower_components',
     }),
+]
+
+if 'djangosaml2' in settings.INSTALLED_APPS:
+    urlpatterns += (
+        url(r'^accounts/', include('djangosaml2.urls')),
+        url(r'^test/', 'djangosaml2.views.echo_attributes'),
+    )
```

Comment:
Yes, most of the time.  There could conceivably be occasions when it might not, though.  

So, since it's working, there's no reason you must change it.  I'm offering the advice that when working with a loosely-typed language, it's better to keep working with the same data type rather than introduce the opportunity for type conversion to take place.